_data-prepper/migrating-from-logstash-data-prepper.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: default
 title: Migrating from Logstash
-nav_order: 30
-redirect_from:
+nav_order: 25
+redirect_from: 
   - /data-prepper/configure-logstash-data-prepper/
 ---
 
 # Migrating from Logstash
 
 You can run Data Prepper with a Logstash configuration.
 
-As mentioned in the [Getting started]({{site.url}}{{site.baseurl}}/data-prepper/get-started/) guide, you'll need to configure Data Prepper with a pipeline using a `pipelines.yaml` file.
+As mentioned in the [Getting started]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/) guide, you'll need to configure Data Prepper with a pipeline using a `pipelines.yaml` file.
```

Comment:
guide --> topic?

Reply:
Let's avoid this by instead saying "As mentioned in [Getting started with Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/), you'll need..."